litellm/types/llms/openai.py

```diff
@@ -401,6 +401,18 @@ class ChatCompletionToolParam(TypedDict):
     function: ChatCompletionToolParamFunctionChunk
 
 
+class Function(TypedDict, total=False):
```

Comment:
It seems that you replicate the types from the openai package? I'm not sure why that's done (versus directly importing from openai), but I copied the types here.